safe/impact_functions/base.py

```diff
@@ -107,6 +102,12 @@
 LOGO_ELEMENT = m.Brand()
 LOGGER = logging.getLogger('InaSAFE')
 
+__author__ = 'akbargumbira@gmail.com'
```

Comment:
Yeah there is some pylon error I think that complains if the imports are not at the top of the file...I am just updating them as I encounter them....